<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_archive</class>
 <widget class="QWidget" name="Edit_archive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ea_save_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ea_cancel_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ea_delete_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="ea_name_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название работы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ea_name_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ea_publisher_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название издательства</string>
   </property>
  </widget>
  <widget class="QLabel" name="ea_date_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дата публикации</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ea_date_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ea_author_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ea_teacher_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ea_publisher_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ea_add_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
